orchestra/utils/notifications.py

```diff
@@ -98,6 +99,32 @@ def notify_status_change(task, previous_status=None):
                   **message_info)
 
 
+def notify_project_status_change(project, previous_status=None):
+    extra_explanation = ''
+    if project.status == Project.Status.PAUSED:
+        status_text = 'paused'
+        extra_explanation = (
+            'All activities will be put on hold until '
+            'the project is reactivated.')
+    elif project.status == Project.Status.ACTIVE:
+        status_text = 'reactivated'
+    elif project.status == Project.Status.ABORTED:
```

Comment:
given you report aborted changes, you could call this in end_project. only if it's easy.